Projects/Checkers/Checkers.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
As a general opening note: Because this is a potential submission to the main branch, everything looks like new code to me so some comments I make might actually be on my original code! 🤓